bzt/jmx/tools.py

```diff
@@ -122,7 +122,8 @@ def modify(self, jmx):
 
             concurrency_list = []
             for group in groups:
-                concurrency_list.append(group.get_concurrency(raw=raw))
+                concurrency = group.get_concurrency(raw=raw)
+                concurrency_list.append(concurrency or 0)
```

Comment:
0 concurrency makes no sense, it will disable this TG by our logic. Shouldn't we default to 1?